frontend/public/components/app-contents.tsx

```diff
@@ -434,6 +434,16 @@ const AppContents_: React.FC<AppContentsProps> = ({ activePerspective }) => {
               }
             />
 
+            <LazyRoute
+              path="/k8s/ns/:ns/:plural/:name/new-snapshot"
```

Comment:
What happens if the user is trying to create a Snapshot when he selects `all-namespaces` ?